Makefile

```diff
@@ -70,6 +70,13 @@ all:
 	$(MAKE) -C $(dir $(BASE_DIR)) build
 
 .PHONY: lint
+# Run lint against the previous commit for PR and branch build
+# In dev setup look at all changes on top of master
+REV := "HEAD^"
+DETACHED := $(findstring "fatal:", $(shell git symbolic-ref HEAD))
```

Comment:
can git symbolic-ref have some fatal messages in case of some other errors than a detached head? I think it would be better to check the return status of git symbolic-ref, since it will return 1 if is not a symbolic ref, and 128 for some other errors.

Reply:
Simplified the check, it did not work in all cases either...